lib/WebSocket.js

```diff
@@ -102,6 +139,11 @@ module.exports = function upgrade(request, socket, head) {
         wsTerminal.emit('connection', ws, request);
       });
       break;
+    case '/app-logs':
```

Comment:
To properly authenticate the WebSocket upgrade request, you'll have to send the request from the CLI to `/api/app-logs`. So that means the case here should be `/api/app-logs`. (See my recent pull request for details.)